src/desktop/apps/auction_support/client/registration_form.coffee

```diff
@@ -13,12 +13,14 @@ module.exports = class RegistrationForm extends ErrorHandlingForm
   events:
     'click .registration-form-content .avant-garde-button-black': 'onSubmit'
     'click .bidding-question': 'showBiddingDialog'
+    'change .registration-form-section__checkbox': 'validateAcceptTerms'
```

Comment:
For clarity, maybe we can call these `Conditions` instead of `Terms` throughout.

Reply:
:+1:!